presto-main/src/main/java/io/prestosql/operator/PagesRTreeIndex.java

```diff
@@ -173,7 +173,7 @@ private static Envelope getEnvelope(OGCGeometry ogcGeometry)
             }
         });
 
-        return matchingPositions.toIntArray(null);
+        return matchingPositions.toIntArray();
```

Comment:
This sounds like separate change? Can you please extract this as separate commit with proper description?

Reply:
ok, I'll create another commit. It's just a cosmetic change as the fastutil marked `toIntArray(object)` deprecated.